Src/Tools/DiscoveryDocPatcher/Patcher.cs

```diff
@@ -13,6 +13,7 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
```

Comment:
Very minor, but I prefer an empty line after the license! And it conforms to most of our sources.
This PR is making changes to this DiscoveryDocPatcher project anyway, so thought I'd include it.